api/v1/helper.go

```diff
@@ -47,6 +49,8 @@ var log = logf.Log.WithName("sriovnetwork")
 // Vendor ID, Physical Function Device ID, Virtual Function Device ID
 var NicIDMap = []string{}
 
+var InitialState SriovNetworkNodeState
```

Comment:
shouldnt this be part of "vars" package ?

also NicIDMap ?

Reply:
can't do it because they you will have a circular import